File: Data Structures & Algorithms/longest-consecutive-sequence/submission-5.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        if(nums.empty()){
            return 0;
        }
        sort(nums.begin(),nums.end());
        int res=0;
        int curr=nums[0];
        int str=0;
        int i=0;
        while(i<nums.size()){
            if(curr!=nums[i]){
                curr=nums[i];
                str=0;
            }
            while(i<nums.size() && nums[i]==curr){
                i++;
            }
            str++;
            curr++;
            res=max(res,str);
        }
        return res;
    }
};

File: Data Structures & Algorithms/remove-element/submission-5.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        ans=[]
        x=0
        for i  in range(len(nums)):
            if nums[i]==val:
                continue
            ans.append(nums[i])
        
        nums[:]=ans
        return len(ans)